app/client/components/delivery/records/deliveryRecords.tsx

```diff
@@ -224,6 +224,20 @@ export const DeliveryRecordsFC = (props: DeliveryRecordsFCProps) => {
           <h1>Delivery history</h1>
         </PageHeaderContainer>
         <PageNavAndContentContainer selectedNavItem={navLinks.subscriptions}>
+          <div
+            css={css`
+              margin: ${space[6]}px 0 ${space[12]}px;
+            `}
+          >
+            <ProductDetailsTable
+              productName={capatalize(
+                props.routeableStepProps.productType.shortenedFriendlyName ||
+                  props.routeableStepProps.productType.friendlyName
+              )}
+              subscriptionId={props.productDetail.subscription.subscriptionId}
+              isGift
```

Comment:
**I think this is always setting it to gift,** rather than taking that from the `productDetail` (you probably want to check the other places where `[GIFT]` is added to see how they work out if the product is a gift)